src/main/java/com/bootcamp/demo/models/Booking.java

```diff
@@ -0,0 +1,78 @@
+package com.bootcamp.demo.models;
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+
+public final class Booking {
+    private UUID id;
+    private UUID serialNumber;
+    private UUID accountId;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    private PaymentStatus payment;
+
+    public Booking() {}
+
+    public Booking(UUID id,UUID serialNumber, UUID accountId,LocalDateTime startDate,LocalDateTime endDate,PaymentStatus payment) {
```

Comment:
There are still some formatting inconsistencies, e.g. missing spaces after some `,`. When multiple persons collaborate on a project, it is easier to have it formatted consistently by all collaborators (for example the default formatting for intellij or a custom one can be used), as it improves readability (long term, you get accustomed to a certain formatting) & reduces the chances of being formatted only later on, when changes are made to this class.
IntelliJ keymap: https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf - includes also the key shortcut for "Reformat code", which will do this automatically.

Reply:
The serial number is a String please. change it.